OpenSim/Simulation/Model/Geometry.h

```diff
@@ -188,13 +187,32 @@ class OSIMSIMULATION_API Geometry : public Component {
     }    //=====================================================================
 };  // END of class Geometry
 
+class OSIMSIMULATION_API DrawingGeometry : public Geometry
+{
```

Comment:
Why is DrawingGeometry and DrawingAppearance necessary. I see a potential explosion of Geometry and Appearance classes with no apparent gain in functionality. Why are we doing this?